velox/exec/HashTable.cpp

```diff
@@ -1712,6 +1712,13 @@ void HashTable<ignoreNullKeys>::prepareJoinTable(
     otherTables_.emplace_back(std::unique_ptr<HashTable<ignoreNullKeys>>(
         dynamic_cast<HashTable<ignoreNullKeys>*>(table.release())));
   }
+  for (int i = 0; i < rows_->columnTypes().size(); ++i) {
```

Comment:
Would you add empty lines before and after this block and a small comment to explain what this does?